index.html

```diff
@@ -0,0 +1,326 @@
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="about-me">
+      <h2 class="head">About<br />Myself</h2>
+      <p class="information">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h4 class="ico">LET’S CONNECT</h4>
+      <ul class="ico-img">
+        <li>
+          <a href="#"
+            ><img class="t-social-media" src="img/twitter.png" alt="twitter"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              class="l-social-media"
+              src="img/linkedin_icon.png"
+              alt="linkedIn"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img class="m-social-media" src="img/medium.png" alt="medium"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img class="g-social-media" src="img/github.png" alt="github"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img class="v-social-media" src="img/vec.png" alt="vector"
+          /></a>
+        </li>
+      </ul>
+      <button type="button" class="resume-btn">Get my resume</button>
+
+      <ul class="lang">
+        <li class="lang-exp">
+          <div class="no-1">
+            <p>Languages</p>
+            <img src="img/arrow_bottom.png" alt="arrow_bottom" />
+          </div>
+          <ul class="languages">
+            <li>
+              <img src="img/js_logo.png" alt="JavaScript" />
+              <a href="#">JavaScript</a>
+            </li>
+            <li>
+              <img src="img/htmlLogo.png" alt="CSS" />
+              <a href="#">HTML</a>
+            </li>
+            <li>
+              <img src="img/css_logo.png" alt="CSS" />
+              <a href="#">CSS</a>
+            </li>
+          </ul>
+        </li>
+        <li class="lang-exp">
+          <div class="no-1">
+            <p>Frameworks</p>
+            <img src="img/arrow_bottom.png" alt="arrow_bottom" />
+          </div>
+          <hr />
+        </li>
+        <li class="lang-exp">
+          <div class="no-1">
+            <p>Skills</p>
+            <img src="img/arrow_bottom.png" alt="arrow_bottom" />
+          </div>
+          <hr />
+        </li>
+      </ul>
+    </section>
+    <footer>
+      <div class="contacts">
+        <div class="contact-intro">
+          <h1 class="contact-me">Contact me</h1>
+      <p class="paragraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+        </div>
+
+      <form
+        class="contact-section"
+        action="https://formspree.io/f/xbjwkev"
+        method="post"
+      >
```

Comment:
- [ ] Kindly ensure you have setup the [form spree ](https://formspree.io/html/) service correctly, when I click submit it shows
`Form not found
Please check the form hashid` 
- Please fix this 🤝